src/app/auth/auth.component.ts

```diff
@@ -201,6 +201,8 @@ export class AuthComponent implements OnInit, OnDestroy {
     const dataRepo: string = this.getDataRepoDetails(sessionInformation);
     this.githubService.storeOrganizationDetails(org, dataRepo);
 
+    this.logger.info("Selected Settings Location: " + sessionInformation);
```

Comment:
```suggestion
    this.logger.info(`Selected Settings Repo: ${sessionInformation}`);
```

Let's use template literals for better readability, and use the phrase "settings repo" to be consistent with our user guide.